.github/workflows/develop-push.yml

```diff
@@ -14,10 +14,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: develop
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
         with:
-          java-version: 1.8
+          java-version: 11
```

Comment:
Please try to keep unrelated changes in separate PRs. Update title and description as appropriate.

Reply:
ah, sorry. The whole repo is just get setup. This is the first chunk code I added, and there are some problems I realized while I add this PR. So I just improve it in this PR. 
Updated PR comment.